docs/content/en/developers/kcp-playground.md

```diff
@@ -0,0 +1,331 @@
+one compute cluster.
+Use the following command to start the playground:
+
+```shell
+kubectl kcp playground start -f test/kubectl-kcp-playground/examples/placement/pool-of-clusters.yaml
+```
+
+This playground gives you a setup similar to the one described in the previous example, but
+both the two kind clusters are reference by a single Location/Placement rule, thus defining 
+a pool of clusters; with this configuration, workloads deployed on this workspace are going
+to be placed only one of the clusters in the pool.
+
+Deploy a workload in KCP:
+
+```shell
+# From another terminal window
+export KUBECONFIG=.kcp/playground.kubeconfig
+
+kubectl kcp playground use shard main
+kubectl ws root:my-org:applications
+kubectl apply -f test/e2e/reconciler/deployment/workloads/deployment.yaml
+kubectl get deploy -A
+```
+
+Check workload have been deployed on one (and only one) compute cluster:
+
+```shell
+kubectl kcp playground use pcluster [us-east1-cluster1|us-east1-cluster2]
+kubectl get deploy -A
+```
+
+Take your time to look at this setup to understand how workload placement can be used in kcp.
+
+### Custom resource synced to a compute cluster
+
+This playground configuration shows how to combine together what we saw about APIBinding and APIExports
+with SyncTargets/Location/Placement rules, and most specifically show cases how to publish APIs as a service provider,
+use them in another workspace and also get them synchronized to a compute cluster.
+
+Use the following command to start the playground:
+
+```shell
+kubectl kcp playground start -f test/kubectl-kcp-playground/examples/apis/cowboy-api-to-pcluster.yaml
+```
+
+This playground gives you the following setup under `root:my-org`:
+- A service-provider workspaces, exposing the cowboys API
+- A infrastructure workspaces with a synctarget, configured to sync the cowboys API, and the corresponding kind cluster.
+- An application workspace biding to the cowboys API and with a placement rule targeting the above sync target.
+
+With this configuration, when a cowboy object is going to be created in kcp, it will be synced to the compute cluster
+as well like it usually happens for deployments.
+
+```shell
+# From another terminal window
+export KUBECONFIG=.kcp/playground.kubeconfig
+
+kubectl kcp playground use shard main
+kubectl ws root:my-org:applications
+kubectl apply -f - <<EOF
+  apiVersion: wildwest.dev/v1alpha1
+  kind: Cowboy
+  metadata:
+    name: woody
+  spec: {}
+EOF
+```
+
+Check workload have been deployed on one (and only one) compute cluster:
+
+```shell
+kubectl kcp playground use pcluster us-east1
+kubectl get cowboys -A
+```
+
+Take your time to look at this setup to understand how API exports and workload placement can be used in kcp.
+
+## Build your own playgrounds
+
+You can build your own playgrounds by starting from one of the above config files
+or defining a new config from scratch using the playground config format described below.
```

Comment:
can't we reference a file with yaml comments and test that automatically? A giant yaml blob here will eventually break.

Reply:
Done!
The test loads the example config file and checks it is valid